lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3315,11 +3323,25 @@ namespace {
       SmallVector<VarDecl *, 4> members;
       SmallVector<FuncDecl *, 4> methods;
       SmallVector<ConstructorDecl *, 4> ctors;
+      SmallVector<TypeDecl *, 4> memberTypes;
```

Comment:
```suggestion
      SmallVector<TypeDecl *, 4> nestedTypes;
```
That's the commonly used Swift terminology.